src/api/courses.json

```diff
@@ -0,0 +1,445 @@
+            "subjectCode": "CSI",
+            "subjectNumber": "2340",
+            "section": "001",
+            "credit": "2",
+            "grade": {
+                "credit": "2",
+                "grade": "N/A",
+                "crn": "43069"
+            },
+            "meetings": [
+                {
+                    "crn": "43069",
+                    "startDate": "2020-09-03T04:00:00.000+0000",
+                    "endDate": "2020-12-15T05:00:00.000+0000",
+                    "startTime": "10:00 AM",
+                    "endTime": "11:47 AM",
+                    "courseType": "In-Person Class Meeting",
+                    "courseTypeCode": "CLAS",
+                    "location": "DH 135",
+                    "campus": "Main Campus",
+                    "meetDays": "Thursday ",
+                    "startHour": 0,
+                    "startMinutes": 0,
+                    "startMonth": 9,
+                    "startYear": 2020,
+                    "startDay": 3,
+                    "startDayOfWeek": 5,
+                    "startWeekOfMonth": 1,
+                    "endHour": 0,
+                    "endMinutes": 0,
+                    "endMonth": 12,
+                    "endYear": 2020,
+                    "endDay": 15,
+                    "endDayOfWeek": 3,
+                    "endWeekOfMonth": 3
+                },
+                {
+                    "crn": "43069",
+                    "startDate": "2020-09-03T04:00:00.000+0000",
+                    "endDate": "2020-12-15T05:00:00.000+0000",
+                    "startTime": "12:00 PM",
+                    "endTime": "1:00 PM",
+                    "courseType": "In-Person Class Meeting",
+                    "courseTypeCode": "CLAS",
+                    "location": "DH 135",
+                    "campus": "Main Campus",
+                    "meetDays": "Thursday ",
+                    "startHour": 0,
+                    "startMinutes": 0,
+                    "startMonth": 9,
+                    "startYear": 2020,
+                    "startDay": 3,
+                    "startDayOfWeek": 5,
+                    "startWeekOfMonth": 1,
+                    "endHour": 0,
+                    "endMinutes": 0,
+                    "endMonth": 12,
+                    "endYear": 2020,
+                    "endDay": 15,
+                    "endDayOfWeek": 3,
+                    "endWeekOfMonth": 3
+                }
+            ],
+            "instructors": [
+                {
+                    "crn": "43069",
+                    "firstName": "Iraida",
```

Comment:
Maybe we should replace these guys with demo data too

Reply:
the names are demo data. i used the name generator website you suggested lol

Reply:
Ha! Nice - they seemed so realistic ;)